src/connectionpools.jl

```diff
@@ -3,6 +3,9 @@ module ConnectionPools
 export Pod, Pool, acquire, release
 
 import Base: acquire, release
+import Sockets
+import MbedTLS
+using ..ConnectionPool: Connection
```

Comment:
i think this makes the modules mutually dependent on each other, which isn't right (see previous comment)

Reply:
i.e. i think `ConnectionPools` should not depend on `ConnectionPool` (which is a specific implementation of the `ConnectionPools` interface)